.dapper

```diff
@@ -85,4 +85,6 @@ fi
 
 [ -t 1 ] && dockerargs="${dockerargs} -t"
 
-docker run -i --rm $(printf -- " -e %s" $DAPPER_ENV) -e "DAPPER_UID=$(id -u)" -e "DAPPER_GID=$(id -g)" -v "${DAPPER_CP}:${DAPPER_SOURCE}${suffix}" ${dockerargs} ${DAPPER_RUN_ARGS} "${container}" "$@"
+# seccomp is unconfined to avoid problems with clone3 in Fedora 35
```

Comment:
> Once we're confident
that all our users have recent enough runtimes, we can re-enable it.

Should we have some inline "remove this when X happens" docs, for our future selves?